source/administration/command-line-tools.rst

```diff
@@ -414,6 +414,7 @@ mattermost export
 
   Child Commands
     -  `mattermost export actiance`_ - Export data from Mattermost in Actiance XML format
+    -  `mattermost export bulk`_ - Export data to a file compatible with the Mattermost Bulk Import format
```

Comment:
Wondering how we can distinguish between TE feature (export bulk) and E20 features (export actiance, export csv, ..)

Reply:
Added version to each command.